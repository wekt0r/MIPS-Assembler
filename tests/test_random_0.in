nor $29, $t1, $1
srl $s6, $t3, 11
addu $7, $10, $31
addi $0, $4, 21679
sltiu $29, $t4, 11217
and $a0, $9, $2
subu $t0, $s7, $3
lui $zero, 10105
srav $a1, $a1, $t4
sltiu $s5, $23, 19929
sltiu $1, $8, 2734
sra $a3, $a3, 6
sltu $s3, $sp, $27
slti $6, $12, 4630
and $3, $a3, $4
div $8, $21
slt $31, $12, $sp
addiu $25, $s1, 1346
add $s7, $20, $t1
andi $14, $s5, 23483
slti $k0, $24, 21133
nor $24, $s6, $t1
div $t5, $28
addu $a2, $22, $0
add $23, $a1, $18
ori $s3, $a2, 29472
xori $gp, $10, 19281
srav $t9, $10, $at
multu $s6, $28
div $t1, $s3
multu $20, $18
nor $7, $s5, $a2
multu $s4, $27
addu $gp, $at, $t4
sllv $s6, $22, $a2
addiu $t1, $t7, 7561
sllv $s0, $23, $t9
andi $s2, $24, 25932
lui $v1, 25945
srl $t2, $4, 16
xor $18, $a1, $at
div $zero, $s4
and $3, $s2, $13
andi $v0, $4, 23828
sra $24, $11, 23
divu $9, $t0
slt $v0, $ra, $3
divu $t2, $4
mflo $t7
sub $s7, $s1, $8
sltu $2, $sp, $25
lui $3, 8701
addi $s3, $23, 22180
ori $a3, $a1, 25099
multu $29, $fp
addi $s0, $zero, 12028
andi $16, $2, 7825
addu $a1, $a0, $s7
xori $a1, $fp, 27886
div $t8, $t2
div $25, $t0
sllv $sp, $fp, $zero
slti $10, $s7, 23858
andi $t6, $t5, 11714
addi $21, $s2, 2259
slti $k0, $a0, 4561
ori $a2, $23, 7760
addu $12, $23, $10
