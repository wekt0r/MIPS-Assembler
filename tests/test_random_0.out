.text
00000000	0121E827	nor	$29,$t1,$1
00000004	000BB2C2	srl	$s6,$t3,11
00000008	015F3821	addu	$7,$10,$31
0000000c	208054AF	addi	$0,$4,21679
00000010	2D9D2BD1	sltiu	$29,$t4,11217
00000014	01222024	and	$a0,$9,$2
00000018	02E34023	subu	$t0,$s7,$3
0000001c	3C002779	lui	$zero,10105
00000020	01852807	srav	$a1,$a1,$t4
00000024	2EF54DD9	sltiu	$s5,$23,19929
00000028	2D010AAE	sltiu	$1,$8,2734
0000002c	00073983	sra	$a3,$a3,6
00000030	03BB982B	sltu	$s3,$sp,$27
00000034	29861216	slti	$6,$12,4630
00000038	00E41824	and	$3,$a3,$4
0000003c	0115001A	div	$8,$21
00000040	019DF82A	slt	$31,$12,$sp
00000044	26390542	addiu	$25,$s1,1346
00000048	0289B820	add	$s7,$20,$t1
0000004c	32AE5BBB	andi	$14,$s5,23483
00000050	2B1A528D	slti	$k0,$24,21133
00000054	02C9C027	nor	$24,$s6,$t1
00000058	01BC001A	div	$t5,$28
0000005c	02C03021	addu	$a2,$22,$0
00000060	00B2B820	add	$23,$a1,$18
00000064	34D37320	ori	$s3,$a2,29472
00000068	395C4B51	xori	$gp,$10,19281
0000006c	002AC807	srav	$t9,$10,$at
00000070	02DC0019	multu	$s6,$28
00000074	0133001A	div	$t1,$s3
00000078	02920019	multu	$20,$18
0000007c	02A63827	nor	$7,$s5,$a2
00000080	029B0019	multu	$s4,$27
00000084	002CE021	addu	$gp,$at,$t4
00000088	00D6B004	sllv	$s6,$22,$a2
0000008c	25E91D89	addiu	$t1,$t7,7561
00000090	03378004	sllv	$s0,$23,$t9
00000094	3312654C	andi	$s2,$24,25932
00000098	3C036559	lui	$v1,25945
0000009c	00045402	srl	$t2,$4,16
000000a0	00A19026	xor	$18,$a1,$at
000000a4	0014001A	div	$zero,$s4
000000a8	024D1824	and	$3,$s2,$13
000000ac	30825D14	andi	$v0,$4,23828
000000b0	000BC5C3	sra	$24,$11,23
000000b4	0128001B	divu	$9,$t0
000000b8	03E3102A	slt	$v0,$ra,$3
000000bc	0144001B	divu	$t2,$4
000000c0	00007812	mflo	$t7
000000c4	0228B822	sub	$s7,$s1,$8
000000c8	03B9102B	sltu	$2,$sp,$25
000000cc	3C0321FD	lui	$3,8701
000000d0	22F356A4	addi	$s3,$23,22180
000000d4	34A7620B	ori	$a3,$a1,25099
000000d8	03BE0019	multu	$29,$fp
000000dc	20102EFC	addi	$s0,$zero,12028
000000e0	30501E91	andi	$16,$2,7825
000000e4	00972821	addu	$a1,$a0,$s7
000000e8	3BC56CEE	xori	$a1,$fp,27886
000000ec	030A001A	div	$t8,$t2
000000f0	0328001A	div	$25,$t0
000000f4	001EE804	sllv	$sp,$fp,$zero
000000f8	2AEA5D32	slti	$10,$s7,23858
000000fc	31AE2DC2	andi	$t6,$t5,11714
00000100	225508D3	addi	$21,$s2,2259
00000104	289A11D1	slti	$k0,$a0,4561
00000108	36E61E50	ori	$a2,$23,7760
0000010c	02EA6021	addu	$12,$23,$10
